controllers/dockermachine_controller.go

```diff
@@ -65,32 +63,8 @@ func (r *DockerMachineReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Watches(
 			&source.Kind{Type: &infrastructurev1alpha2.DockerMachine{}},
```

Comment:
This needs to be watching a cluster-api Machine

Reply:
Thank you. I have fixed it.